core/src/testils/proptest/arbitrary.rs

```diff
@@ -13,3 +27,12 @@ impl Arbitrary for MacAddr {
         any::<[u8; 6]>().prop_map_into()
     }
 }
+
+impl Arbitrary for Mbuf {
+    type Parameters = ();
+    type Strategy = BoxedStrategy<Self>;
+
+    fn arbitrary_with(_args: Self::Parameters) -> Self::Strategy {
```

Comment:
thought about using params here, but went against it for now.

Reply:
don't need prop_compose. you can just do, should work

```
LazyJust::new(|| Mbuf::new().unwrap()).boxed()
```

Reply:
maybe an even simpler one

```
impl Arbitrary for Mbuf {
    type Parameters = ();
    type Strategy = fn() -> Self;

    fn arbitrary_with(_args: Self::Parameters) -> Self::Strategy {
        || Mbuf::new().unwrap()
    }
}
```